tests/phpunit/tests/blocks/serialize.php

```diff
@@ -78,4 +80,27 @@ public static function add_attribute_to_inner_block( $block ) {
 		}
 		return $block;
 	}
+
+	/**
+	 * @ticket 59327
+	 *
+	 * @covers ::traverse_and_serialize_blocks
+	 *
+	 * @dataProvider data_serialize_identity_from_parsed
+	 *
+	 * @param string $original Original block markup.
+	 */
+	public function test_traverse_and_serialize_identity_from_parsed( $original ) {
+		$blocks = parse_blocks( $original );
+
+		$actual   = traverse_and_serialize_blocks(
+			$blocks,
+			function ( $block ) {
+				return $block;
+			}
+		);
+		$expected = $original;
+
+		$this->assertSame( $expected, $actual );
```

Comment:
```suggestion
		$actual = traverse_and_serialize_blocks(
			$blocks,
			function ( $block ) {
				return $block;
			}
		);

		$this->assertSame( $original, $actual );
```

No need to add variable use existing one.

Reply:
I copied it over from the same test for `serialize_blocks`. Happy to update in both places.